src/main/java/com/ldtteam/blockout/controls/Scrollbar.java

```diff
@@ -144,17 +144,17 @@ public void drawSelf(final int mx, final int my)
     }
 
     @Override
-    public void handleClick(final int mx, final int my)
+    public boolean handleClick(final double mx, final double my)
     {
         if (getContentHeightDiff() <= 0)
         {
-            return;
+            return true;
```

Comment:
Not sure about return value. Does that mean scrollbar is not visible - should we then allow clicking in lower layers under scrollbar?

Reply:
Yes, in this case there is no scrollbar, so we should allow clicks at that location (either to drag, or for a button, or for whatever).